### ISSUE TITLE
Set startup projects

### ISSUE THREAD

**Issue by dave-mcalister** [tunnelvisionlabs/DebugCommandLine#3]
It would be really usefult to be able to configure the command line options together with the visual studio startup projects.

**Comment by sharwell** [comment]
I agree. When I was working on game development our debugging environment was controlled by the startup command line arguments. I've found that working on Visual Studio extensions is the opposite - the command line rarely changes but the startup executable frequently does.
